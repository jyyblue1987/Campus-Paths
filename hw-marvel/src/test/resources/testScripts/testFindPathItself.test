# Test if the output is correct for finding a path to itself

LoadGraph superhero staffSuperheroes.csv

FindPath superhero Ernst-the-Bicycling-Wizard Ernst-the-Bicycling-Wizard