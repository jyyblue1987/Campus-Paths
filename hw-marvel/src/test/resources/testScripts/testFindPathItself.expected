# Test if the output is correct for finding a path to itself

loaded graph superhero

path from Ernst-the-Bicycling-Wizard to Ernst-the-Bicycling-Wizard: